### PR TITLE
doc: document the workaround to install a non-latest ScyllaDB version

### PR DESCRIPTION
**Pull request by annastuchlik**
This PR is related to https://github.com/scylladb/scylla-enterprise/issues/2176.
It adds a FAQ about a workaround to install a ScyllaDB version that is not the most recent patch version.
In addition, the link to that FAQ is added to the patch upgrade guides 2021 and 2022 .

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -228,6 +228,32 @@ Scylla comes with its own version of the Apache Cassandra client tools, in the p
...
+   $ cat <<EOF | sudo tee /etc/apt/preferences.d/99scylla-enterprise
+   Package: scylla-enterprise*
+   Pin: version 2021.1.0-0.20210511.9e8e7d58b-1
+   Pin-Priority: 1001
+   EOF
+
+Pinning may be particularly useful when you want to downgrade ScyllaDB or upgrade to a version that is not the latest 
+available version.
+
+See `this topic <https://unix.stackexchange.com/questions/350192/apt-get-not-properly-resolving-a-dependency-on-a-fixed-version-in-a-debian-ubunt>`_
```
Comment (tarzanek) [inline docs/faq.rst:253]: well not sure if we should cross ref to non official docs(that just state what is said here), below ubuntu community is enough imho

also above is a convenience, so other option is to name ALL packages with their respective version, which is tedious and error prone (but our upgrade guides do use it for revert)

Comment (annastuchlik) [reply docs/faq.rst]: Thanks, I've applied your suggestions. 
I've removed the link and added the alternative solution suggested by @avi in [this comment](https://github.com/scylladb/scylla-enterprise/issues/2176#issuecomment-1095041407), as it's less complicated than listing all the dependencies with full names.

**Review by tarzanek — APPROVED** [review]
lgtm
thank you!

File: docs/faq.rst [hunk 2]
```diff
@@ -228,6 +228,37 @@ Scylla comes with its own version of the Apache Cassandra client tools, in the p
...
+   EOF
+
+Pinning may be particularly useful when you want to downgrade ScyllaDB or upgrade to a version that is not the latest 
+available version. See `this article <https://help.ubuntu.com/community/PinningHowto>`_ for details about pinning on Debian-based systems.
+
+Alternatively, you can explicitly install **all** the ScyllaDB packages for the desired non-latest version. For example:
+
+.. code-block:: console
+
+   apt-get install scylla-enterprise{,-server,-jmx,-tools,-tools-core,-kernel-conf,-node-exporter,-conf,-python3}=2021.1.0-0.20210511.9e8e7d58b-1
```
Comment (tarzanek) [inline docs/faq.rst:257]: sudo missing here

**Review by vladzcloudius — COMMENTED** [review]
Looks good except for item @tarzanek commented about (the `sudo`).

**Comment by annastuchlik** [comment]
Thanks! I've added "sudo" to the command.

**Comment by annastuchlik** [comment]
@eliransin ping

**Review by eliransin — APPROVED** [review]
LGTM

**Comment by annastuchlik** [comment]
@scylla/scylla-maint Please merge.

**Pull request merged**